Makefile

```diff
@@ -195,16 +195,27 @@ CPU_COUNT = $(shell cat /proc/cpuinfo | grep processor | wc -l )
 	@docker-compose --env-file .env --file services/docker-compose.yml --file services/docker-compose.local.yml --log-level=ERROR config > $@
 
 .stack-ops.yml: .env $(docker-compose-configs)
+	# Compiling config file for filestash
+	$(eval TMP_PATH_TO_FILESTASH_CONFIG=$(shell set -o allexport; \
+	source $(CURDIR)/.env; \
+	set +o allexport; \
+	python3 scripts/filestash/create_config.py))
 	# Creating config for ops stack to $@
-	@docker-compose --env-file .env --file services/docker-compose-ops.yml --log-level=ERROR config > $@
+	# -> filestash config at $(TMP_PATH_TO_FILESTASH_CONFIG)
+	@$(shell \
+		export TMP_PATH_TO_FILESTASH_CONFIG="${TMP_PATH_TO_FILESTASH_CONFIG}"; \
```

Comment:
use ```&&``` instead of ```;``` it's a better practice. else if one fail it will not be discovered.